compiler/src/dotty/tools/dotc/typer/Typer.scala

```diff
@@ -4013,6 +4013,8 @@ class Typer(@constructorOnly nestingLevel: Int = 0) extends Namer
           cpy.Select(qual)(pre, name.toTypeName)
         case qual: This if qual.symbol.is(ModuleClass) =>
           cpy.Ident(qual)(qual.symbol.name.sourceModuleName.toTypeName)
+        case _ =>
+          errorTree(tree, em"cannot convert to $tree to an instance creation expression")
```

Comment:
```suggestion
          errorTree(tree, em"cannot convert from $tree to an instance creation expression")
```
I think there is a small typo here